src/reducers/toolbar.js

```diff
@@ -0,0 +1,16 @@
+const INITIAL_STATE = {
+    isModalActive : false
+}
+
+const toolbarReducer = (state = INITIAL_STATE, action) => {
+    switch (action.type) {
+        case 'TOGGLE_MODAL':
```

Comment:
If this is the only thing this reducer is doing (and nothing will be added to it later) we can probably just move this stuff into local component useState to handle the modal -- if more will be added to it later, disregard this

Reply:
This reducer is not used and has been deleted.